pkgs/applications/misc/blugon/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python3, libX11, libXrandr }:
+
+stdenv.mkDerivation rec {
+  pname = "blugon";
+  version = "1.11.4";
+
+  src = fetchFromGitHub {
+    owner = "jumper149";
+    repo = "blugon";
+    rev = "c53b47eac7f9b23adeb62f8df10f38ffbc0dd156";
```

Comment:
```suggestion
    rev = version;
```

Reply:
Ok, I should've checked the nixpkgs manual when you said I should use the version tag. Done.